platform/projects_and_organizations/members.mdx

```diff
@@ -0,0 +1,76 @@
+---
+id: members
+title: Members
+sidebar_label: Members
+---
+
+Once you have created a project and set up the languages, you can add members to your project. Members are the people who will be translating your project.
+
+## Adding members
+
+To add members to your project, navigate to the "Members" page. Then click on the "Invite user" button.
+You can add a member by **entering their email address or creating an invitation link** to share with them.
+
+Then you can select the **permissions** you want to give the member. The permissions are explained in the [next section](#permissions).
+
+When you add a member to your project, the **member is automatically added to your organization**.

```

Comment:
This is confusing. The user is added to the organization just informatively, but it's actually not a member.